auslib/web/common/history.py

```diff
@@ -0,0 +1,107 @@
+import json
```

Comment:
Yes, this was my understand, that admin will reuse some endpoints. Is not right?